web-app/CKS.Web/Startup.cs

```diff
@@ -42,10 +45,34 @@ public void ConfigureServices(IServiceCollection services)
 			}
 
 			services.AddControllers();
+
+			if (!CurrentEnvironment.IsDevelopment())
+			{
+				services.AddHsts(options =>
+				{
+					//TODO: Investigate why https://hstspreload.org/ times out for nice.org.uk currently
+					options.Preload = true;
+					options.IncludeSubDomains = true;
+					options.MaxAge = TimeSpan.FromDays(60);
+				});
+
+				services.AddHttpsRedirection(options =>
+				{
+					options.RedirectStatusCode = StatusCodes.Status308PermanentRedirect;
+					options.HttpsPort = 443;
+				});
+			}
 		}
 
 		public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 		{
+			if (!env.IsDevelopment())
+			{
+				app.UseExceptionHandler("/Error");
+				app.UseHttpsRedirection();
+				app.UseHsts();
```

Comment:
I think we'll need to use [`UseForwardedHeaders` here](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-3.1#forwarded-headers-middleware-order)? As we're behind an ALB and nginx